Topic8/src/main/java/com/aguirre/topic8/services/CourseService.java

```diff
@@ -0,0 +1,40 @@
+package com.aguirre.topic8.services;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.aguirre.topic8.models.Course;
+import com.aguirre.topic8.models.Student;
+import com.aguirre.topic8.repositories.CourseDao;
+import com.aguirre.topic8.repositories.StudentCourseDao;
+
+@Service
+public class CourseService {

```

Comment:
Every Service (like the DAOs) need to have an interface and the class that implement it